sdcm/nemesis.py

```diff
@@ -832,11 +828,11 @@ def disrupt_terminate_and_replace_node(self):  # pylint: disable=invalid-name
         InfoEvent(message='StartEvent - Terminate node and wait 5 minutes').publish()
         self._terminate_and_wait(target_node=self.target_node)
         InfoEvent(message='FinishEvent - target_node was terminated').publish()
-        new_node = self._add_and_init_new_cluster_node(old_node_ip, rack=self.target_node.rack)
+        new_node = self._init_new_node_with_latency_calculation(old_node_ip, rack=self.target_node.rack)
```

Comment:
You should not change the name of the current methods in order to measure latency, it should be implemented differently (decorator approach for example)

Reply:
there was a duplicated function, i just removed the duplication out of our way